RestructuredText/syntax_test_restructuredtext.rst

```diff
@@ -21,6 +21,22 @@ This is not a comment
  These are still part of the comment if they're indented.
 .. <- comment.line.double-dot.restructuredtext
 
+
+<<<<<<< HEAD
+.. <- meta.block.conflict.begin.diff punctuation.section.block.begin.diff
+..^^^^^ meta.block.conflict.begin.diff punctuation.section.block.begin.diff
+..     ^ meta.block.conflict.begin.diff - entity - punctuation
+..      ^^^^ meta.block.conflict.begin.diff entity.name.section.diff
+..          ^ meta.block.conflict.begin.diff - entity - punctuation
+
```

Comment:
Did you leave out `========` intentionally?

Reply:
This is one of the ambiguities we are facing in RST and MD. 

This line is already tested as part of a heading in line 48.